src/Components/Users/Users.tsx

```diff
@@ -127,62 +129,69 @@ export default function Users() {
-                <th scope="col">Phone Number</th>
-                <th scope="col">Email</th>
-                <th scope="col">Date Created</th>
-                <th scope="col">Action</th>
-
-              </tr>
-            </thead>
-            <tbody>
-              {userList.map((user) => (
-
-                <tr key={user.id} >
-
-                  <td >{user.userName}</td>
-                  <td >
-                    {user.isActivated ?
-                      (
-                        <div className={`${style.active} ${style.badgeActiveInActive}`}>Active</div>) : (
-                        <div className={`${style.inactive} ${style.badgeActiveInActive}`}>InActive</div>
-                      )}
-                  </td>
-                  <td >{user.phoneNumber}</td>
-                  <td >{user.email}</td>
-                  <td >{new Date(user.group.creationDate).toLocaleDateString()}</td>
-                  <td>
-
-                    <button className={`${style.blockBtn} mx-2`} onClick={() => toggleActivationStatus(user.id, user.isActivated)}>
-                      {user.isActivated ? 'Block' : 'Unblock'}
-                    </button>
-                    <i onClick={() => showViewModel(user.id)}
-                      className="fa-solid fa-eye text-success"></i>
-                  </td>
-                </tr>
 
-              ))}
-            </tbody>
-          </table>
-        </div>
-        :
-        <NoData />
+            <table className="table">
+              <thead className='table-head table-bg'>
+                <tr>
 
-      }
+                  <th scope="col">user Name</th>
+                  <th scope="col">Statues</th>
+                  <th scope="col">Phone Number</th>
+                  <th scope="col">Email</th>
+                  <th scope="col">Date Created</th>
+                  <th scope="col">Action</th>
+
+                </tr>
+              </thead>
+              <tbody>
+                {userList.map((user) => (
+
+                  <tr key={user.id} >
+
+                    <td >{user.userName}</td>
+                    <td >
+                      {user.isActivated ?
+                        (
+                          <div className={`${style.active} ${style.badgeActiveInActive}`}>Active</div>) : (
+                          <div className={`${style.inactive} ${style.badgeActiveInActive}`}>InActive</div>
+                        )}
+                    </td>
+                    <td >{user.phoneNumber}</td>
+                    <td >{user.email}</td>
+                    <td >{new Date(user.group.creationDate).toLocaleDateString()}</td>
+                    <td>
+
+                      <button className={`${style.blockBtn} mx-2`} onClick={() => toggleActivationStatus(user.id, user.isActivated)}>
+                        {user.isActivated ? 'Block' : 'Unblock'}
+                      </button>
+                      <i onClick={() => showViewModel(user.id)}
```

Comment:
We can make an icon clickable by adding onClick event to it, but this is not a good practice. and we are losing a lot of features by not using a button for that like using tap no navigate between interactive elements, and pressing space to click it, and more.

You should instead make a button and nest the icon inside it, and the button should be giving the onClick event.

below is an illustration of some of the accessibility issues that are caused by not using a button:

before (the view icon is not accessible via the tap key nor it can be clicked using space):
![before using a button](https://github.com/Rawan-Mohamed/project-management-system-1/assets/64327685/13bdb3b0-1cca-45e7-89b3-20d8e67ac573)

after (the view icon is  accessible via the tap key nor it can be clicked using space):
![after using a button](https://github.com/Rawan-Mohamed/project-management-system-1/assets/64327685/c73eec57-2a8a-49c9-a804-c8bfad338a06)